test/bids_examples_eeg_rest_fmri.jsonld

```diff
@@ -118,7 +118,7 @@
     {
       "@id": "_:000010",
       "@type": "https://openminds.ebrains.eu/core/FileRepository",
-      "IRI": "file:///home/peyman-user/Desktop/data/bids_test/bids-examples/eeg_rest_fmri",
+      "IRI": "file:///home/peyman-user/Desktop/Code/bids-examples/eeg_rest_fmri",
```

Comment:
I'm confused. Why is this a local path? should this not be a path on GitHub?

Reply:
I agree we should fix this, but let's do that in a separate pull request